sql/flaky/tests.sql

```diff
@@ -15,12 +15,12 @@ suites AS (
     JOIN suites s ON cr.check_suite_id = s.id
 )
 SELECT
-    a.title AS test_name
-  , count(*) AS count
+    regexp_replace(a.title, '([._\\`*_{}\[\]\(\)<>:|"''#$])', '\\$1') AS "Test name"
```

Comment:
Add a comment explaining what this regexp matches. 
Why does the underscore occur twice?

Reply:
`g` is not needed because `regexp_replace` in Trino always replaces all occurrences and it doesn't take the fourth argument: https://trino.io/docs/current/functions/regexp.html

Added a comment and removed the extra underscore.

Reply:
Sorry, I confused this with `benchmark-reports` project which runs queries on PostgreSQL